civis/io/_tables.py

```diff
@@ -807,6 +938,8 @@ def callback(future):
             return
         else:
             url = outputs[0]["path"]
-            return _download_file(url, filename)
+            file_id = outputs[0]["file_id"]
+            log.info('Exported results to Civis file %s', file_id)
```

Comment:
I think this should be a debug-level emit.

Reply:
done